network-config-analyzer/nca.py

```diff
@@ -12,7 +12,7 @@
 from RESTServer import RestServer
 
 
-def _valid_path(path_location, allow_ghe=False, allow_k8s=False, allow_calico=False):
+def _valid_path(path_location, allow_ghe=False, allow_k8s=False, allow_calico=False, allow_istio=False):
```

Comment:
let's not have a separate flag for each platform. Enough to have `allow_k8s` and `allow_other_platforms`.

Reply:
changed a bit differently, since it seems we need differentiation for calico or k8s only in the case of pods lists.